src/steps/writing/creation/createESLintRC.ts

```diff
@@ -200,20 +200,15 @@ module.exports = {
 				? ""
 				: `"jsdoc",
 		`
-		}${options.excludeTests ? "" : `"no-only-tests",`}${
-			options.excludeLintPerfectionist ? "" : `"perfectionist",`
-		}${options.excludeLintRegex ? "" : `"regexp",`}${
-			options.excludeTests ? "" : `\n"vitest",`
-		}
+		}${options.excludeLintPerfectionist ? "" : `"perfectionist",`}${
+			options.excludeLintRegex ? "" : `"regexp",`
+		}${options.excludeTests ? "" : `\n"vitest",`}
 	],
 	reportUnusedDisableDirectives: true,
 	root: true,
 	rules: {
 		// These off/less-strict-by-default rules work well for this repo and we like them on.
-		"@typescript-eslint/no-unused-vars": ["error", { caughtErrors: "all" }],${
-			options.excludeTests ? "" : `\n"no-only-tests/no-only-tests": "error",`
-		}
-
+		"@typescript-eslint/no-unused-vars": ["error", { caughtErrors: "all" }],
```

Comment:
[Bug] This newline is used in the repo's ESLint config, so it'll need to be present here too.

```suggestion

		"@typescript-eslint/no-unused-vars": ["error", { caughtErrors: "all" }],
```